pkg/crc/services/dns/dns.go

```diff
@@ -124,5 +124,6 @@ func addOpenShiftHosts(serviceConfig services.ServicePostStartConfig) error {
 	return adminhelper.UpdateHostsFile(serviceConfig.IP, serviceConfig.BundleMetadata.GetAPIHostname(),
 		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift"),
 		serviceConfig.BundleMetadata.GetAppHostname("console-openshift-console"),
+		serviceConfig.BundleMetadata.GetAppHostname("downloads-openshift-console"),
 		serviceConfig.BundleMetadata.GetAppHostname("default-route-openshift-image-registry"))
```

Comment:
This seems unrelated to `Also clean hosts file on Windows`

Reply:
Yes extracted to a new commit.